lib/workflows/send_shutoff_vm_email.py

```diff
@@ -0,0 +1,174 @@
+from typing import List, Optional, Union
+
+from email_api.emailer import Emailer
+from openstack_query import ServerQuery, UserQuery
+
+from enums.cloud_domains import CloudDomains
+from enums.query.props import ServerProperties
+from enums.query.query_presets import QueryPresetsGeneric
+
+from structs.email.email_params import EmailParams
+from structs.email.email_template_details import EmailTemplateDetails
+from structs.email.smtp_account import SMTPAccount
+
+
+def find_servers_with_shutoff_vms(
+    cloud_account: str, from_projects: Optional[List[str]] = None
+):
+    """
+    :param cloud_account: string represents cloud account to use
+    :param from_projects: A list of project identifiers to limit search in
+    """
+
+    # Find VMs that have been in shutoff state for more than 60 days
+    server_query = ServerQuery()
+    server_query.where(
+        QueryPresetsGeneric.ANY_IN, ServerProperties.SERVER_STATUS, values=["SHUTOFF"]
+    )
+    server_query.run(
+        cloud_account,
+        as_admin=False,
```

Comment:
change this to `True` otherwise it won't work for all projects!
You'll need to adjust the tests too!

Reply:
Thank you, fixed this in [dbf8dbb](https://github.com/stfc/st2-cloud-pack/pull/234/commits/dbf8dbbf26170e73359102d52d84c80efc08c96e)